aws/resource_aws_ami_test.go

```diff
@@ -218,30 +265,35 @@ func testAccCheckAmiEbsBlockDevice(bd *ec2.BlockDeviceMapping, ed *ec2.EbsBlockD
 	}
 }
 
-func testAccAmiConfig_basic(rName string) string {
+func testAccAmiConfig_base(size int, rName string) string {
```

Comment:
Super small nit since there's another feedback item below: We try to keep the randomized name as the first test configuration function parameter where possible -- in this context it also keeps it consistent with the other functions in this file.

```suggestion
func testAccAmiConfig_base(rName string, size int) string {
```